docs/getting-started.rst

```diff
@@ -0,0 +1,74 @@
+===============
+Getting Started
+===============
+
+Installing Prerequisite Software
+--------------------------------
+
+SALib requires `NumPy <http://www.numpy.org/>`_, `SciPy <http://www.scipy.org/>`_,
+and `matplotlib <http://matplotlib.org/>`_ installed on your computer.  Using
+`pip <https://pip.pypa.io/en/stable/installing/>`_, these libraries can be
+installed with:
+
+::
+
+    pip install numpy
+    pip install scipy
+    pip install matplotlib
+    
+Installing SALib
+----------------
+
+To install the latest stable version of SALib using pip, run the following
+command:
+
+::
+
+    pip install SALib
+    
+To install the latest development version of SALib, run the following
+commands.  Note that the development version may be unstable and include bugs.
+We encourage users use the latest stable version.
+
+::
+
+    git clone https://github.com/SALib/SALib.git
+    cd SALib
+    python setup.py install
```

Comment:
You could change this to `python setup.py develop` which installs a smiling to the github repository instead of a hard install (which would be out of date as soon as you changed anything.